src/main/java/com/talentpath/shamazin/showItemPage/services/InfoService.java

```diff
@@ -0,0 +1,46 @@
+package com.talentpath.shamazin.showItemPage.services;
+
+import com.talentpath.shamazin.showItemPage.daos.InfoRepository;
+import com.talentpath.shamazin.showItemPage.daos.ItemRepository;
+import com.talentpath.shamazin.showItemPage.exceptions.NoSuchItemException;
+import com.talentpath.shamazin.showItemPage.models.Info;
+import com.talentpath.shamazin.showItemPage.models.Item;
+import org.springframework.beans.BeanUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class InfoService {
+    @Autowired
+    InfoRepository infoDao;
+
+    public List<Info> getAllInfo() {
+        return infoDao.findAll();
+    }
+
+    public Info addInfo(Info info) {
+        return infoDao.saveAndFlush(info);
+    }
+
+    public void deleteInfo(Integer id) {
+        infoDao.deleteById(id);
+    }
+
+    public Info getInfo(Integer id) throws NoSuchItemException {
+        Optional<Info> info = infoDao.findById(id);
+        if(info.isPresent()) return info.get();
+        else throw new NoSuchItemException("No item with id: " + id);
```

Comment:
Do we have cases where an item is present with no info description? If so, NoSuchItemException and this message might not give an accurate representation of that error. If not, then we can just ignore my feedback